hummingbot/strategy/fixed_grid/fixed_grid_order_tracker.pyx

```diff
@@ -0,0 +1,49 @@
+from typing import (
+    Dict,
+    List,
+    Tuple
+)
+
+from hummingbot.core.data_type.limit_order cimport LimitOrder
+from hummingbot.core.data_type.limit_order import LimitOrder
+from hummingbot.connector.connector_base import ConnectorBase
+from hummingbot.strategy.market_trading_pair_tuple import MarketTradingPairTuple
+from hummingbot.strategy.order_tracker cimport OrderTracker
+
+NaN = float("nan")
+
+
+cdef class FixedGridOrderTracker(OrderTracker):
```

Comment:
the method added seems to already be available in the OrderTracker class. I think it may be better to refer to OrderTracker if there is no difference.

Reply:
Thanks. I've now added the missing types, and used the existing market_status_data_frame and OrderTracker. I recompiled and tested the code to make sure everything works as before.

Reply:
Thanks. Nothing else from me. although a bit weird to see the fix to bitfinex connector and avell strategy in this pr as it is not part of the scope of this pr.

Reply:
I had to make those fixes to get the code to compile, so I could test it. I can remove those changes for the sake of this merge if necessary.

Reply:
Hi, I received information that any new functionality submitted will need to include the necessary unit tests to provide the code coverage. So you have to remove the line added in .coveragerc and add the unit test required in order for this strategy to be merged to the codebase. You may refer to the tests folder on how to add the unit tests.
Also, please resolve the merge conflict in the pr. Thank you.

- Tech Review DAO